Updated READMEs and docs to be better and match B9 1.3.1

ZakaHaceCosas opened the pull request:
Summary of the changes made
- Updated HTML++, CSS 3.25 and Luau manuals to match current standard.
- Also updated user manuals.
- Formatted Markdown files.
- Also, updated README.md files of some of the repo's folders.
- Featured an extension from @KashTheKing which looks cool enough (WebX related, obviously...).
- Added a small section for custom DNS.
- Featured the WebX Plus' custom DNS in there.
- Removed recommendations from browser list, and instead added a simple OS compatibility card.
- Added a recommendation for JS boys (reminder that Lua starts arrays at 1 and not 0).
- Fixed many typos.
- Fixed all mistakes in documentation, like `css_properties` being underscored instead of being `css-properties` dashed, or `set_content` being misspelled as `set_contents`, among others.

Note:
- The browser's UI got updated, but the images didn't, so please someone screenshot Napture in his PC and update the images (don't ask me to do it, Bussin Napture doesn't work on my Gaming OS 10 PC 😃)

Fixes #197 

If there's anything wrong, any feedback, whatever... just tell me and I'll happily help. Good afternoon.

ZakaHaceCosas committed:
added a small disclaimer for JS users

ZakaHaceCosas committed:
correct header organisation

ZakaHaceCosas committed:
fix

ZakaHaceCosas committed:
put it into a block

ZakaHaceCosas committed:
logos made by me :D

ZakaHaceCosas committed:
maybe like this?

ZakaHaceCosas committed:
maybe this way

ZakaHaceCosas committed:
MAYBE THIS WAY?

ZakaHaceCosas committed:
forget it, will find another way

ZakaHaceCosas committed:
fix a typo

ZakaHaceCosas committed:
better formatting and that kind of stuff

ZakaHaceCosas committed:
better readmes

ZakaHaceCosas committed:
Merge branch 'face-hh:master' into master

ZakaHaceCosas committed:
started works on 1.3.0

ZakaHaceCosas committed:
progress (not done with lua yet but commit because yes)

ZakaHaceCosas committed:
cool

ZakaHaceCosas committed:
Merge branch 'face-hh:master' into master

ZakaHaceCosas committed:
version bump

ZakaHaceCosas commented:
Just updated all texts to say 1.3.1 instead of 1.3.0.

ZakaHaceCosas committed:
Fix misusage of underscores instead of dashes.

ZakaHaceCosas commented:
Alright fixed the underscoring issue, this should now be OK for merging

ZakaHaceCosas committed:
Fixed typos with set_content function

ZakaHaceCosas commented:
Fixed some places saying "`set_contents`" instead of "`set_content`", which is the correct spelling of the function.

ZakaHaceCosas committed:
removed some misinformation (based on issue BUG:  Misinformation in readme #197)

ZakaHaceCosas commented:
Based on #197 , made a small update (basically remove one line).

RealPacket commented:
Add "Fixes #197" in your PR description so when the PR is merged then #197 will be closed.

ZakaHaceCosas commented:
Updated the comment 👍

ZakaHaceCosas committed:
small fix

ZakaHaceCosas committed:
Fix an error with GET /domains

ZakaHaceCosas commented:
Fixed an error - `GET` /domains did not take `amount` but `page_size` as the parameter to define the number of domains per page. Now it's corrected.

ZakaHaceCosas committed:
fixed typos

ZakaHaceCosas commented:
Fixed a few typos / mistakes and reworded some things.

inventionpro commented:
> Fixed an error - `GET` /domains did not take `amount` but `page_size` as the parameter to define the number of domains per page. Now it's corrected.

it also allows `size` and `limit` to do the same

ZakaHaceCosas committed:
better browser table and custom dns

ZakaHaceCosas committed:
sligh clarification about DNS custom

ZakaHaceCosas commented:
- Removed the "recommended" field from browser list as at the end it's an opinion.
- Removed the unnecessary spacing of the Markdown in the browser table.
- Added OS compatibility for browsers, being +++ perfection, ++ working, and + working with many known issues (or a single one reported by many, just as #50 - therefore the Windows+ for Napture). If my criteria doesn't look right, just change it anyway, lol.
- Added a very small section dedicated to custom DNS. Featured WebX+. Also added a clarification to the registrar list.

ZakaHaceCosas commented:
If anything else needs to be changed, improved, added, or whatever, just tell me, but I think this is _finally_ ready to get merged.

ZakaHaceCosas committed:
Fix forgotten button tag
